examples/monetization-with-stripe/pages/api/payment-handler.ts

```diff
@@ -0,0 +1,47 @@
+import { buffer } from "micro";
+import { storage } from "utils/storage";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import Stripe from "stripe";
+
+export const PAID_COOKIE_TITLE = "paid-for-[miro-integration-name]";
```

Comment:
maybe we should rename this to reflect what it is better, eg: `payment_storage_key`